src/test/resources/augmentation-tests.adoc

```diff
@@ -786,7 +1028,7 @@ type Person4 {
 
 type Person5 implements HasMovies {
   id: ID!
-  movies(first: Int, offset: Int): [Movie]
+  movies(first: Int, offset: Int): [Movie] @relation(direction : OUT, from : "from", name : "LIKES", to : "to")
```

Comment:
from and to are defaults, OUT is also default

Reply:
this is just copied from what the SchemaWriter prints out